pastel/client.go

```diff
@@ -136,7 +136,17 @@ func (client *client) RegTicket(ctx context.Context, regTxid string) (RegTicket,
 	return ticket, nil
 }
 
-func (client *client) GetBlockCount(ctx context.Context) (int64, error) {
+func (client *client) GetBlockVerbose1(ctx context.Context, blkHeight int32) (*GetBlockVerbose1Result, error) {
+	result := &GetBlockVerbose1Result{}
+
+	if err := client.callFor(ctx, result, "getblock", fmt.Sprint(blkHeight), "1"); err != nil {
+		return result, errors.Errorf("failed to get block: %w", err)
+	}
+
+	return result, nil
+}
+
+func (client *client) GetBlockCount(ctx context.Context) (int32, error) {
```

Comment:
Return type should be int64 because `res.GetInt()` return `int64`. We may risk truncate value here.